package.json

```diff
@@ -63,7 +63,6 @@
     "@nx/eslint": "17.2.8",
     "@nx/jest": "17.2.8",
     "@nx/workspace": "17.2.8",
-    "@prettier/sync": "^0.5.0",
```

Comment:
_heck yes_ async everywhere

Reply:
everywhere *except* `eslint-plugin-internal`. One day we'll have async plugins . . . .

Reply:
🙏